pkg/security/probe/probe.go

```diff
@@ -1553,6 +1552,13 @@ func NewProbe(config *config.Config, statsdClient statsd.ClientInterface) (*Prob
 	return p, nil
 }
 
+func boolTouint64(value bool) uint64 {
```

Comment:
nit: we could maybe move this function to a common package like utils ?

Reply:
yes I will do that